hs_access_control/models.py

```diff
@@ -395,18 +401,20 @@ def delete_group(self, this_group):
     @property
     def view_groups(self):
         """ 
-        Get a list of groups accessible to self for view.
+        Get a list of active groups accessible to self for view. Inactive groups will be included only if self owns
```

Comment:
As I understood the group owner should be able turn on and off the 'active' status. This functionality has been implemented at the UI level.